client/src/utils/fight/utils/findFighter.ts

```diff
@@ -17,4 +20,13 @@ const findFighter = (
   stepFighter?: number,
 ) => (typeof stepFighter === 'undefined' ? undefined : fighters.find((f) => f.index === stepFighter));
 
+export const findHUDFocusedFighter = (
+  fightFighters: Fighter[],
+  fighters: AnimationFighter[],
+  team: string,
+) => {
+  const animationFighter = fighters.find((f) => f.HUDFocused && f.team === team);
+  return fightFighters?.find((f) => f.id === animationFighter?.id);
```

Comment:
fightFighters is not declared optional above
```suggestion
  return fightFighters.find((f) => f.id === animationFighter.id);
```